packages/rum-core/src/domain/rumEventsCollection/view/trackInitialViewTimings.ts

```diff
@@ -67,6 +68,7 @@ export function trackNavigationTimings(lifeCycle: LifeCycle, callback: (timings:
           domContentLoaded: entry.domContentLoadedEventEnd,
           domInteractive: entry.domInteractive,
           loadEvent: entry.loadEventEnd,
+          firstByte: entry.responseStart,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** isn't [`TTFB = responseStart - navigationStart`](https://developer.mozilla.org/en-US/docs/Glossary/time_to_first_byte) and not simply `responseStart`?

Reply:
Right, but "`navigationStart`" is `0` 🙂  (or would be 0 if it was set in the PerformanceTiming entry)

Reply:
I suspected that might be the case 😅 was just double checking